packages/flex-plugins-api-toolkit/src/scripts/createConfiguration.ts

```diff
@@ -138,6 +139,7 @@ export default function createConfiguration(
           description: plugin.description,
           changelog: version.changelog,
           isPrivate: installedPlugin.private,
+          isArchived: plugin.archived || version.archived,
```

Comment:
then for this case, wouldn't you only need `version.archived` too?

Reply:
Technically yes, but given we are making two separate requests, one to plugins and one to versions, there could be a diff between two requests. Since we already have both plugins and versions available, there is no harm of doing an || here. In the previous case, `plugin` was not available.

Reply:
that seems verrry unlikely but given that the data is already there that's fine 😛